src/pages/People.js

```diff
@@ -0,0 +1,407 @@
+import React from "react"
+import Helmet from "react-helmet"
+import s from "styled-components"
+import { Grid, Cell } from "styled-css-grid";
+import "../pages/styles.scss"
+import { Title, Text, Subtitle} from "../components/shared/Typography"
+import { Navbar, Footer } from "../components"
```

Comment:
prefer imports by specificity so

React
packages (styled components is more general than react helmet which is more general than styled-css-grid...etc) (this is up for debate)

imports from components
imports from subdirs of components
imports from subdirs of subdirs of components

stylesheets

images